std/zip.d

```diff
@@ -293,6 +293,10 @@ final class ZipArchive
     import std.conv : to;
     import std.datetime.systime : DosFileTime;
 
+    enum endOfCentralDirRecordLength = 22;
```

Comment:
It might be simpler declaring this data structure as a D struct. Then this would be its `sizeof`.

Reply:
Is planed to be done in a later PR. Maybe even a union can be used to avoid those calles of getUshort/getUint/getUlong.